### ISSUE TITLE
Feature Request: refresh browser windows when the file changes

### ISSUE THREAD

**Issue by jnoortheen** [django-extensions/django-extensions#1609]
most of the JS frameworks support this and it aids development. 
Can we add the same feature to `runserver_plus` ?

I have seen https://github.com/harikvpy/django-autorefresh but that is not maintained for a long time. Also it requires installing browser extension. is there any way we can reload the browser/client session ? we could use a simple JS script to poll a hash/datetime and reload if that changed. We could insert this JS file during DEBUG to templates as well.

**Comment by trbs** [comment]
I am happy to look at Pull Requests for this feature. (That said I think we would need to be careful with excepting something like this and make sure its solid and maintainable)

Was thinking it would make sense to put this into `Django Debug Toolbar` as a panel or something... and low and behold... https://github.com/gregmuellegger/django-debug-toolbar-autoreload but that has also been abandoned...

**Comment by jnoortheen** [comment]
Yes gulp seems to be used like done [here](https://stackoverflow.com/questions/39914533/gulp-and-running-django-server-with-logging)
